### ISSUE TITLE
Accounts with 0 ether generated with ganache

### ISSUE THREAD

**Issue by moda20** [ConsenSys-archive/ganache-cli-archive#533]
* I asked about this on [gitter](https://gitter.im/ConsenSys/truffle): [ ]

## Expected Behavior

Accounts generated by `ganache-cli` should have 100 ether in them.

## Current Behavior

the account (coinbase) has 0 ether after running this command 
`ganache-cli --port 22001 --mnemonic --networkId 100  --db /root/ganachedb`

## Your Environment
* Version used: Ganache CLI v6.1.0 
* Version of Truffle/Remix/Other tools used: Truffle v4.1.8
* NodeJS Version: [ ] 6.x, [ ] 7.x (unsupported), [x] 8.x, [ ] 9.x
* Operating System and version (include distro if Linux): OSX high Sierra

**Comment by mikeseese** [comment]
Thanks for reporting this! I haven't had the chance to reproduce this yet, but I'm guessing the lack of providing a `mnemonic` after specifying the flag is causing the issue

**Comment by mikeseese** [comment]
I'm not seeing this behavior in `v6.1.6`; I'm guessing this got fixed in `v6.1.5`. Below is the output I'm receiving:

```
ganache-cli --port 22001 --mnemonic --networkId 100 ~/ganachedb
Ganache CLI v6.1.6 (ganache-core: 2.1.5)

Available Accounts
==================
(0) 0x057e9718f7c14aa3dd054b92ad8a6b58e44abe97 (~100 ETH)
(1) 0xceb5802fcc9512f77a04dc2dbca035f98200f0a2 (~100 ETH)
(2) 0xfdd1baf494d956618cb5d3a315cf528a5af3cfc2 (~100 ETH)
(3) 0xeea8f8041197cbfc4d4f49ba34b471ace2770153 (~100 ETH)
(4) 0x11477acfaaa17ff038b88305d9ae2c445c8f0883 (~100 ETH)
(5) 0x667aa93d83a1440e26a73bcf4f27b8ef57e60b14 (~100 ETH)
(6) 0x425a77c9d792d1a97c9a66abac5313e89408258c (~100 ETH)
(7) 0x4d7facefefe5d23ac520d826c8961fc0bd9d6ad3 (~100 ETH)
(8) 0x90bdd08e695c61b223c40be2003ac730a255154e (~100 ETH)
(9) 0x73fe2d80731cab07e2fd229849c5f3bb6c90b84e (~100 ETH)
```
